### ISSUE TITLE
2.9.3 is broken

### ISSUE THREAD

**Issue by Elrondo46** [vkohaupt/vokoscreenNG#2]
Not registering settings, ~/.vokoscreenNG/vokoscreenNG.conf contains just the version and nothing about settings or window position.

It seems there is a segfault in pure Gnome environment.

I know this is a beta version it can be buggy and requires bug reports

Thanks in advance

**Comment by Elrondo46** [comment]
Just not this is the release version inside your git

**Comment by AlfredoRamos** [comment]
> It seems there is a segfault in pure Gnome environment.

Can you debug it with GDB? I maintain this application in the AUR, I'm interested if this is a dependency issue, because I haven't been able to reproduce it.

```shell
gdb -ex run vokoscreen
```

When the segfault occurs, just type `bt`.

The new beta (2.9.4) generates the configuration file at `~/.config/vokoscreen/vokoscreen.conf` still the same issue though.

**Comment by AlfredoRamos** [comment]
I do not contribute in the development of this application, I only pointed out that without that back trace it would be harder to determine what the issue is.

I didn't quite understand why anybody would "laugh about Arch for just one app", but now that you mentioned Manjaro, this problem might be related to that distro. Manjaro is not Arch even when the former is derived from the latest.

I tested it with Cinnamon, GNOME and Mate without errors.
